mountpoint-s3/tests/fuse_tests/lookup_test.rs

```diff
@@ -177,7 +177,12 @@ where
 
     // Not really a "lookup" test, but since we're playing with Unicode, may as well do it here
     let new_path = mount_point.path().join("🐈/👻");
-    let mut f = OpenOptions::new().write(true).create(true).open(&new_path).unwrap();
+    let mut f = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .truncate(false)
```

Comment:
nit: why use `truncate(false)` here? Mountpoint accepts it but does not really support it. Shouldn't we use `truncate(true)` most of the time, or (better yet?) `create_new`?

Reply:
For this PR, I just wanted to fix clippy errors and the default value when we don't specify them is false. So, I'm only matching the behavior.

Reply:
>the default value when we don't specify them is false

I think I'd lean on setting `truncate` to false throughout the tests for this reason, except of course where we explicitly test truncate behaviour.

Or `create_new` works too in this test, since it is equivalent. 😄